tests/QuestionSetTest.php

```diff
@@ -35,6 +35,24 @@ public function testCreateQuestionSet()
         }
     }
 
+    public function testCreateQuestionSetWithTimeLimit()
+    {
+        $person = self::createTestPerson();
+        $qs = $person->question_sets->create(17);
+
+        $this->assertSame($qs->time_limit, 17);
+    }
+
+    public function testTimeLimitExpired()
+    {
+        $person = self::createTestPerson();
+        $qs = $person->question_sets->create(2);
+        sleep(4);
```

Comment:
Yes - we are proving that a short time limit expires and the client receives the expired flag properly set as NOW expired. I made it short, but it still adds briefly to the test run. But it confirms that it is returned AND it is accessible to the client code.